issuance/issuance.go

```diff
@@ -44,21 +44,14 @@ type ProfileConfig struct {
 	AllowSCTList    bool
 	AllowCommonName bool
 
-	Policies            []PolicyInformation `validate:"omitempty,dive"`
+	Policies            []PolicyConfig `validate:"min=1,dive"`
```

Comment:
Nitpick, but what if instead of jumping into a new struct with a field, this  field becomes
```
PolicyOIDs []string `validate:"required,min=1"`
```

and we could use the String method on asn1.ObjectIdentifiers.

Reply:
I strongly considered doing exactly this. Then I got brain-overwhelmed at the idea of doing a config schema change (rewrite this change to conditionally use either config field, then get SRE to switch which field they use, then land another change to get rid of the old field) when I have this many other things going on and decided to keep it like this.

Happy to do the schema simplification if folks prefer, though.

Reply:
How about just a boulder issue for the future to that exactly that?

Reply:
FWIW, this sounds like a lot of config churn for not a lot of benefit, since we expect this field to always contain exactly one OID, `2.23.140.1.2.1`.

If we wanted to do some config cleanup here I would be more excited about deprecating the Policies config field entirely. Boulder is and always will be a BR-compliant CA. It does and always will do DV, not OV or EV. So we can hardcode the single OID `2.23.140.1.2.1`.

Reply:
I like this idea, but would prefer to do it as a follow-up.